core/templates/dev/head/dashboard/Dashboard.js

```diff
@@ -50,25 +61,55 @@ oppia.controller('Dashboard', [
       $window.location = '/create/' + explorationId;
     };
 
+    $scope.openExploration = function(status, explorationId) {
+      if (status === 'private') {
+        $scope.showExplorationEditor(explorationId);
+      } else {
+        var isOpen = !$scope.isDropdownOpen[explorationId];
+        $scope.isDropdownOpen = {};
+        $scope.isDropdownOpen[explorationId] = isOpen;
+      }
+    };
+
     $scope.myExplorationsView = 'card';
     $scope.setMyExplorationsView = function(viewType) {
       $scope.myExplorationsView = viewType;
     };
 
+    $scope.isMobileView = function() {
```

Comment:
Just a note, we do have functions elsewhere starting with "is". If we wanted to have a variable then we would do something like $scope.viewIsMobile.

On the other hand, I'm seeing functions called isMobileView() and checkForMobileView(). That's a bit confusing, particularly the second one which seems to have side-effects. Maybe the latter should be something like updatePublishTextGivenScreenWidth()?